inc/container.class.php

```diff
@@ -1635,19 +1635,19 @@ public static function preItemUpdate(CommonDBTM $item)
     {
         self::preItem($item);
         if (array_key_exists('_plugin_fields_data', $item->input)) {
-            $data = $item->input['_plugin_fields_data'];
-            //update data
-            $container = new self();
-            if (
-                count($data) == 0
-                || $container->updateFieldsValues($data, $item->getType(), isset($_REQUEST['massiveaction']))
-            ) {
-                $item->input['date_mod'] = $_SESSION['glpi_currenttime'];
-
-                return true;
+            foreach ($item->input['_plugin_fields_data'] as $containerClass) {
+                $data = $containerClass;
+                //update data
+                $container = new self();
+                if (
+                    count($data) == 0
+                    || $container->updateFieldsValues($data, $item->getType(), isset($_REQUEST['massiveaction']))
+                ) {
+                    $item->input['date_mod'] = $_SESSION['glpi_currenttime'];
```

Comment:
You can `exit` from `foreach` here, no need to check for all container if one of them already trigger an update on `date_mod`